### ISSUE TITLE
The Index needs a container that can be styled

### ISSUE THREAD

**Issue by rjsparks** [rfc-format/draft-iab-html-rfc-bis#1]
The index is neither contained nor is it given an id.

The h2 for toc is given an id of "toc" (and things inside are given a class of "toc")

We need the same for the index.

**Comment by hildjj** [comment]
Agree.  The index also needs to be wrapped in a section like the TOC, as in #2.

```html
<section id="index">
    <h2>Index</h2>
    <div class="index">
...
</section>
```

Note that the TOC already has a reference to `#index` in the prototype, without a corresponding `id="index"` anywhere in the doc.  Bug.
